modules/standard/BigInteger.chpl

```diff
@@ -2212,6 +2212,18 @@ module BigInteger {
 
 
   // divexact
+
+/*
+
+Computes the quotients when corresponding elements of a bigint array *n* are divided by corresponding elements of a bigint array *d*, and stores the quotient in another bigint array preceding the function call for whichever elements of the array are exactly divisible, with remainder zero.
+
+* Arguments:
+    * const ref n: bigint: a bigint array
+    * const ref d: bigint: another bigint array
```

Comment:
Since this is a common thing to say in these documentation comments, chpldoc has direct support for it, see https://chapel-lang.org/docs/tools/chpldoc/chpldoc.html#argument-and-return-yield-types

So for example you could write:

   :arg n: a bigint
   :type n: bigint

but these will be redundant with what the documentation already displays in my opinion.

Reply:
There is also
```
 :returns: n/d provided that d divides n
```

Reply:
It would be important to say that the result is stored in `this` which is also referred to as "the method receiver".